### ISSUE TITLE
Question: What is the recommend way to add helpers now? 

### ISSUE THREAD

**Issue by rodw** [linkedin/dustjs#181]
### The TL/DR Version

Now that `dust.helpers` has been removed from linkedin/dustjs what is the recommended way to add helper functions?  

Do I _need_ dustjs-helpers?  

Is safe or recommend to call `dust.helpers = {}` when `helpers` is missing?  

Would it be better/safer/clearer to keep an empty `helpers` attribute in the core object?
### The Long Version

Previously to add a custom helper function to dust one could add a property to the `dust.helpers` object.  For example, to add the custom tag `{@foo}{/foo}` one could write something like:

``` javascript
dust.helpers.foo = function() { return "This is foo." }
```

or 

``` javascript
dust.helpers['foo'] = function() { return "This is foo." }
```

As part of the refactoring of code over to dustjs-helpers, the `dust.helpers`  was removed from the core  linkedin-dustjs.  This makes the code above fail (obviously).

Via the unit tests in the dustjs-helpers repo I see that one can replace:

``` javascript
var dust = require('dustjs-linkedin');
```

with:

``` javascript
var dust = require('dustjs-helpers');
```

which then includes `dust.helpers`, but that also brings in the full suite of helpers in dustjs-helpers (i assume).

By inspection and some trial and error it seems that I can also add `dust.helpers` "manually", e.g.,:

``` javascript
dust = require('linkedin-dustjs');
if (typeof dust.helpers === "undefined" || dust.helpers === null) {
  dust.helpers = {};
}
```

and everything seems to work out OK (as far as I've tested at least).  I haven't  been spelunking  in dustjs-helpers to check, but I assume that's more or less what dustjs-helpers is doing as well.

Is that the recommended way to do it?

Also, although currently the base dust object checks for the presence of `dust.helpers` and seems to handle things more or less as-before, is that a "contract" or a coincidence?  Are there tests to protect that behavior (other than what's found in dustjs-helpers, I guess)?  Might it better or safer or clearer to simply have an keep an `dust.helpers` attribute in the core rather than having extension libraries (of which there may be more than one adding helpers) creating the attribute independently?
